internal/guiconfig/config.go

```diff
@@ -100,28 +102,54 @@ func ParseOverridesForEnv(env, v string) (map[string]interface{}, error) {
 	return overrides, nil
 }
 
-// Environment takes an environment name and returns default values for
-// services on that environment.
-func environment(env string) (map[string]string, error) {
-	switch env {
-	case "production":
-		return nil, nil
-	case "staging":
-		return environmentValues("https://api.staging.jujucharms.com"), nil
-	case "qa":
-		return environmentValues("https://www.jujugui.org"), nil
-	}
-	return nil, fmt.Errorf("invalid environment: %q", env)
+// Environments holds a map of environment names to their corresponding info.
+var Environments = map[string]env{
+	"production": env{
+		ControllerAddr: "jimm.jujucharms.com:443",
+	},
+	"staging": env{
+		ControllerAddr: "jimm.staging.jujucharms.com:443",
+		overrides:      envOverrides("https://api.staging.jujucharms.com"),
+	},
+	"brian": env{
```

Comment:
While some of us know what this means, I think it's probably best to have this as an easter egg and maintain the "qa" option. :stuck_out_tongue: